### ISSUE TITLE
Shorthand CRUD methods.

### ISSUE THREAD

**Issue by g105b** [PhpGt/Database#20]
When calling a query shorthand, infer the CRUD method from the start of the query name:

+ getById = retrieve
+ createNewUser = create
+ etc.

**Comment by g105b** [comment]
Longhand example:

```php
$resultSet = $databaseClient["collectionName"]->query("getById", ["id" => 1]);
```

Shorthand example:

```php
$resultSet = $databaseClient["collectionName"]->getById(["id" => 1]);
```

+ Method starting with "create" or "insert" will return a string ID.
+ Method starting with "get" or "retrieve" will return a `ResultSet`.
+ Method starting with "set" or "update" will return an integer number of affected rows.
+ Method starting with "remove" or "delete" will return an integer number of affected rows.

**Comment by g105b** [comment]
Closed in preference of #51
